package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "ab-testing-tool-ubuntu",
+  "version": "1.0.0",
+  "description": "npm ubuntu dependencies",
+  "main": "index.js",
+  "dependencies": {
+    "jasmine-core": "^2.2.0",
+    "karma": "^0.12.31",
+    "karma-jasmine": "^0.3.5",
+    "karma-phantomjs-launcher": "^0.1.4"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "kevinfan",
+  "license": "ISC",
```

Comment:
@kyfantaz is there a particular reason you're using an ISC license? Can you clarify?